### DISCUSSION TITLE
New feed is useless.

### DISCUSSION THREAD

**Discussion by slembcke** [community/community#37533] [Other Feature Feedback, Questions, & Ideas]
Is this new dash page the same as the beta social media feed thing GitHub kept enabling by default months ago? It's utterly useless. It used to give me information I wanted or needed to know for my own projects or people working in my field. Now it's just utterly full of garbage from people I don't know, fields I don't work in, and languages I don't use.

**Comment by jevinskie** [comment]
Where are the damn commits and pushes??

**Comment by biggestsonicfan** [comment]
I'm not seeing a single project I actively watch or have pushed code to on my feed. It's all recommended and showing users star other repos when I don't even follow that user...

**Reply by jevinskie** [reply]
Or suggesting I follow people I already follow.

**Reply by biggestsonicfan** [reply]
Oh that's an oof

**Comment by jevinskie** [comment]
This is what I see:

https://user-images.githubusercontent.com/187848/198347327-452769ad-ad0f-46d0-ac4f-66a978fbb4f3.MOV

**Reply by pawamoy** [reply]
Yeah, after filtering I have only a handful of items in my feed...

**Comment by capjamesg** [comment]
+1. I do not enjoy the new feed experience. I no longer find the feed useful.
